docs/source/examples/QartodTestExample_netCDF.ipynb

```diff
@@ -430,17 +517,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Explore results\n",
+    "# Explore results: qc test variables are named [variable_name]_qartod_[test_name]\n",
     "out_c = xr.open_dataset(outfile_c)\n",
-    "print(out_c)"
+    "out_c"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Aggregate/rollup flag\n",
+    "out_c[\"qartod_qc_rollup\"]"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python (ioosqc38)",
    "language": "python",
-   "name": "python3"
+   "name": "ioosqc38"
```

Comment:
I usually leave the generic python3 name there to avoid this kind of error.
Jupyter notebooks can be a pain :-/

```suggestion
   "name": "python3"
```

Reply:
this is the absolute worst to have to change this manually everytime I change the notebook...